plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -82,14 +80,7 @@ class ServiceDestroyModal extends React.Component {
   }
 
   shouldForceUpdate() {
```

Comment:
We no longer need any force related commands/checks as it only applies when nested services are in deploying state. We can remove this method.

Reply:
Couldn't a single service/pod be in deploying state? I'm pretty sure I needed this once with a pod

Reply:
Oh my mistake - you are correct. I thought this was a group delete modal.